cmake/RunReverseDD.cmake

```diff
@@ -0,0 +1,201 @@
+# These need to be defined by the caller
+# SOURCE_DIR
+# BINARY_DIR
+# ENERGYPLUS_EXE
+# IDF_FILE
+# BUILD_FORTRAN
+# TEST_FILE_FOLDER
+# ENERGYPLUS_FLAGS
+# PYTHON_EXECUTABLE
+
+get_filename_component(IDF_NAME "${IDF_FILE}" NAME_WE)
+get_filename_component(IDF_EXT "${IDF_FILE}" EXT)
+get_filename_component(EXE_PATH "${ENERGYPLUS_EXE}" PATH)
+
+# Create path variables
+set(OUTPUT_DIR_NAME "testfiles_ReverseDD")
```

Comment:
When this test runs, it will create a new `testfiles_reverseDD` directory in the build tree, and run EnergyPlus, resulting in the following structure:

```
testfiles_reverseDD/
├─ 1ZoneUncontrolled/
│  ├─ Regular/
│  │  ├─ eplusout.csv
│  │  ├─ eplusmtr.csv
│  ├─ Reversed/
│  │  ├─ eplusout.csv
│  │  ├─ eplusmtr.csv
```